optaweb-tsp-ui/src/components/Location.tsx

```diff
@@ -14,12 +14,21 @@
  * limitations under the License.
  */
 
-import PropTypes from 'prop-types';
-import React from 'react';
+import * as React from "react";
```

Comment:
According to the currently used code style (Airbnb), strings must use single quotes. This should have been reported by ESLint. If ESLint is not compatible with `.tsx` files we need to switch to another linter but some kind of automatic linting must be kept in place.

This might be helpful https://intellij-support.jetbrains.com/hc/en-us/community/posts/115000225170/comments/115000301124

Reply:
@yurloc can you check that on IntelliJ all works as expected now.

vscode works as expected

Reply:
What do you mean "works as expected"? We have single quotes in the rest of the files and double quotes here, which is an inconsistency regardless how I configure my IDE :slightly_smiling_face:. This should be automatically detected by TSLint or Prettier by running `npm run lint`.

Reply:
Enforced in tslint.json
```json
    "quotemark": { "options": "single", "severity": "error" },
```